### DISCUSSION TITLE
初回起動以降は`npm start`で問題ない旨を記載してほしい

### DISCUSSION THREAD

**Discussion by dodo-lab** [ws-4020/mobile-app-crib-notes#475] [Ideas]
学習終盤まで、`npm run android`で起動していました。
学習の実装に入る前に、初回起動以降は`npm start`で問題ない旨を記載しておけば、効率よく進められそうです。

**Comment by creature-water-valley** [comment]
以下のPRでネイティブに変更がなければ`npm run [android/ios]`の実行は初回のみで良いという補足を追加
- https://github.com/ws-4020/mobile-app-crib-notes/pull/1294/files#diff-97c64833fe086fe565015037e6547f0e7b19c214af259ab854ab4c0ed3858bcbR69

**Comment by creature-water-valley** [comment]
対応済みなのでクローズ
